api/tools/src/cron/cleanupOrphanedWorkspaces.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
```

Comment:
please add comment that this is intended for local dev convenience

Reply:
I believe that all scripts in this directory are intended to be used that way, so I added a "README.md" to the directory that explains what these scripts are for.